app/worker/main.py

```diff
@@ -95,25 +107,39 @@ def create_sheet_task(self, sheet_json: str):
                 stats["errors"].append(str(e))
 
     except SystemExit as e:
-        log_error(e, f"create_sheet_task: SystemExit from AA")
+        log_error(e, "create_sheet_task: SystemExit from AA")
 
     if stats["archived"] > 0:
         with get_db() as session:
-            worker_crud.update_sheet_last_url_archived_at(session, sheet.sheet_id)
+            worker_crud.update_sheet_last_url_archived_at(
+                session, sheet.sheet_id
+            )
 
     logger.info(f"SHEET DONE {sheet=}")
     # TODO: is this used anywhere? maybe drop it
-    return schemas.CelerySheetTask(success=True, sheet_id=sheet.sheet_id, time=datetime.datetime.now().isoformat(), stats=stats).model_dump()
+    return schemas.CelerySheetTask(
+        success=True,
+        sheet_id=sheet.sheet_id,
+        time=datetime.datetime.now().isoformat(),
+        stats=stats,
+    ).model_dump()
+
 
+def get_orchestrator_args(
+    group_id: str, orchestrator_for_sheet: bool, cli_args: list = None
+) -> list:
+    if cli_args is None:
+        cli_args = []
```

Comment:
```console
app/worker/main.py:129:67: B006 Do not use mutable data structures for argument defaults
    |
128 | def get_orchestrator_args(
129 |     group_id: str, orchestrator_for_sheet: bool, cli_args: list = []
    |                                                                   ^^ B006
130 | ) -> list:
131 |     aa_configs = []
    |
    = help: Replace with `None`; initialize within function
```

Reply:
this does not feel like a useful ruff suggestion unless I'm missing something, can we skip/disable it?

Reply:
It looks like it's something that is somewhat useful to do: 
> The same mutable object is then shared across all calls to the function. If the object is modified, those modifications will persist across calls, which can lead to unexpected behavior.

> Instead, prefer to use immutable data structures, or take None as a default, and initialize a new mutable object inside the function body for each call.

I think we'll generally be fine, but it looks like if it was linked to a specific list, then it could, in theory, start with a non-empty list if we added things to it every time we ran the function.

Link: https://docs.astral.sh/ruff/rules/mutable-argument-default/